src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1299,6 +1305,12 @@ int xcldev::device::bandwidthKernelXbtest(void)
 {
     std::string output;
 
+    if (sensor_tree::get<std::string>("system.linux", "N/A").find("Red Hat") != std::string::npos) {
+        std::cout << "Testcase not supported on Red Hat. Skipping validation"
+                  << std::endl;
+        return -EOPNOTSUPP;
+    }
+
```

Comment:
we probably should not change the xbtest part. if we have a xbtest binary works on ppc, both verify kernel and bandwidth should all be able to run.

Reply:
oh right. I'll remove that